Fix for comp imports

Sizzzles opened the pull request:
Comp importing should work again. Added early level ups, updated items for set 8 including augments, fixed screen cords for pickup rounds and augment position

Sizzzles committed:
level up in early rounds

Sizzzles committed:
Fix misspelled item and augment position update

Sizzzles committed:
Fixed warnings

Sizzzles committed:
Updated for set 8

Sizzzles committed:
Comp importing works again

jppuzzle commented:
would it be possible to add an update option to get the most updated comps or do you have to manually clear the cache?

Sizzzles commented:
The current way I am doing this is by `del /s /q <TFT-OCR-BOT DIRECTORY>\cached_data\cached8.json` in a batch file

Mine looks like this
![S4T5I9H](https://user-images.githubusercontent.com/75231345/210473259-6b15b87a-b179-4203-a041-a2490a462518.png)

Kiruscha commented:
> Loading champions and comps...
> Loading from web... (It can take up to ~1 min)
> Traceback (most recent call last):
>   File "C:\Users\Me\Desktop\TFT-OCR-BOT-main jan2023\main.py", line 31, in <module>
>     auto_comps.LoadChampionsAndComps(comps_manager)
>   File "C:\Users\Me\Desktop\TFT-OCR-BOT-main jan2023\auto_comps.py", line 200, in LoadChampionsAndComps
>     lol_chess_comps = __LoadLolChessComps(response_meta_comps, set_current, comp_manager)
>   File "C:\Users\Me\Desktop\TFT-OCR-BOT-main jan2023\auto_comps.py", line 85, in __LoadLolChessComps
>     json_parsed = json.loads(text_proceed)
>   File "C:\Users\Me\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
>     return _default_decoder.decode(s)
>   File "C:\Users\Me\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
>     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
>   File "C:\Users\Me\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 353, in raw_decode
>     obj, end = self.scan_once(s, idx)
> json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 18 column 1 (char 218240)

Yes, please implement Sizzles fix.

Sizzzles committed:
Checks items more frequently and added more augments

Sizzzles committed:
Added more level ups

STAGE  2-1     LEVEL  4
STAGE: 2-5     LEVEL  5
STAGE: 3-2     LEVEL  6
STAGE: 4-1     LEVEL  7
STAGE: 5-1     LEVEL  8

Sizzzles committed:
Exits game after victory instead of waiting

Sizzzles committed:
Augment reroll and spacing cleanup

Sizzzles committed:
Clears anvils after PvE rounds

Sizzzles committed:
Delay for champion moves and Pillow ver update

Sizzzles committed:
Update screen_coords.py

Sizzzles committed:
updated screen_coords.py

Sizzzles committed:
Updated item pickup for units put  on far right

Sizzzles committed:
Victory continue button actually works now

Sizzzles committed:
Added victory exit fuction, Should work now

Sizzzles committed:
Faster accept on queues, should pickup all orbs

Sizzzles committed:
Actually selects augments from list

Sizzzles committed:
Update arena.py

Sizzzles committed:
Update screen_coords.py

Sizzzles committed:
Changed league path back to default

Sizzzles committed:
Fixed Augments not being picked at 2-1

Sizzzles committed:
Updated coords for comps such as underground

Sizzzles committed:
Item placement now happens after anvil rounds

Sizzzles committed:
Update README.md

Sizzzles committed:
Code Cleanup

Sizzzles committed:
Fixed Auto Comps

Sizzzles committed:
auto comps rewrite || New requirements

Sizzzles committed:
Added planned feature

Sizzzles committed:
Dynamic augments testing

Sizzzles committed:
Dynamic augments testing

Sizzzles committed:
Update README.md

Sizzzles committed:
Dynamic and user defined augments

Sizzzles committed:
Dynamic and user defined augments

Sizzzles committed:
Merge branch 'testing' of https://github.com/Sizzzles/TFT-OCR-BOT into testing

Sizzzles committed:
Dynamic and user defined augments

Auto comps now have augments for each comp. If augment isn't found defaults to user defined augment then if still not found defaults to first augment position

Sizzzles committed:
Delete cached_data directory
